content/react/concepts/components/components.md

```diff
@@ -13,8 +13,8 @@ CatalogContent:
 
 In React, components make up the UI of the application. Components are pieces of code that are reuseable and independent. Each component comes with 1 or 2 of the following things:
 
-- `state`: Data in a component expected to change over time
-- `props`: Data that can be shared from parent component to child component
+- [state](https://www.codecademy.com/resources/docs/react/state) : Data in a component expected to change over time 
+- [props](https://www.codecademy.com/resources/docs/react/props) : Data that can be shared from parent component to child component
```

Comment:
```suggestion
- A [`state` object](https://www.codecademy.com/resources/docs/react/state) that contains component data that is expected to change over time. 
- A [`props` object](https://www.codecademy.com/resources/docs/react/props) with data that can be passed down from parent component to child component.
```

Let's keep the back-ticks around "state" and "props" and treat these list items as complete sentences.